Src/Common/W3C/W3CUtilities.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace Microsoft.ApplicationInsights.Common.W3C
+{
+    using System;
+    using System.ComponentModel;
+    using System.Globalization;
+
+    /// <summary>
+    /// W3C distributed tracing utilities.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class W3CUtilities
```

Comment:
Do we need this public? Removal will be hard once we ship in stable.

Reply:
sorry, file was added by mistake